cibyl/outputs/cli/ci/system/impls/base/colored.py

```diff
@@ -39,4 +40,27 @@ def print_system(self, system, indent=0):
         if self.verbosity > 0:
             printer[-1].append(f' (type: {system.system_type.value})')
 
+        if self.query in (QueryType.FEATURES_JOBS, QueryType.FEATURES):
+            for feature in system.features.values():
+                printer.add(self.print_feature(feature), 1)
+
+            if self.query == QueryType.FEATURES:
+                # if the user did not ask for jobs, just print the features
+                # and return
+                return printer.build()
```

Comment:
I think this 'if' does nothing, both branches lead to the same output

Reply:
Right, thanks!